src/context/userStore.ts

```diff
@@ -403,6 +409,30 @@ export const useUserStore = create<UserState>()(
             showError('Transfer', e)
           }
         },
+        claimTokens: async () => {
+          const user_address = get().address
+          const handleOp = get().handleOp
+          const showError = useModalStore.getState().showError
+          const step = useModalStore.getState().step
+
+          step('Claiming Teia DAO tokens', 'Claiming Teia DAO tokens', true)
+
+          // Download the distribution file mapping file
+          const mapping = await downloadFileFromIpfs(DISTRIBUTION_MAPPING_IPFS_PATH)
+
+          // Download the Merkle data
+          const merkleDataPath = MERKLE_DATA_IPFS_PATHS[mapping[user_address]]
```

Comment:
Doesn't handle the possibility that the user_address is not in the mapping dictionary. That could happen with new users

Reply:
Fixed that in my last commits